include/nighthawk/common/request_source_plugin_config_factory.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "envoy/api/api.h"
+#include "envoy/common/pure.h"
+#include "envoy/config/typed_config.h"
+
+#include "nighthawk/common/request_source.h"
+
+namespace Nighthawk {
+
+struct RequestSourceContext {
+  RequestSourceContext(RequestSourceContext& context)
+      : api(std::move(context.api)), header(std::move(context.header)) {}
+  RequestSourceContext(Envoy::Api::ApiPtr api_input, Envoy::Http::RequestHeaderMapPtr header_input)
+      : api(std::move(api_input)), header(std::move(header_input)) {}
+  Envoy::Api::ApiPtr api;
+  Envoy::Http::RequestHeaderMapPtr header;
+};
+using RequestSourceContextPtr = std::unique_ptr<RequestSourceContext>;
```

Comment:
nit: missing space. I wonder, if you run `ci/do_ci.sh fix_format`, does it inject this space for you?

Reply:
Calling ci/do_ci.sh fix_format just now didn't seem to do anything on this file.